.travis.yml

```diff
@@ -1,3 +1,6 @@
+env:
+  - GO111MODULE=on
+
 language: go
 go:
   - 1.12.x
```

Comment:
I think we can consider adding 1.13.x here

Reply:
I think we should stay with 1.12 for now, 1.13 has been just released. We should wait and see if there are no major problems reported by go community.